superset/viz.py

```diff
@@ -363,24 +363,34 @@ class TableViz(BaseViz):
     credits = 'a <a href="https://github.com/airbnb/superset">Superset</a> original'
     is_timeseries = False
 
+    def should_be_timeseries(self):
+        fd = self.form_data
+        # TODO handle datasource-type-specific code in datasource
+        return (
+            (fd.get('granularity') and fd.get('granularity') != 'all') or
```

Comment:
code below says: `self.form_data.get("granularity") == "all"` and here u return `fd.get('granularity') != 'all'`